packages/react-ui/internal/themes/BasicLightTheme.ts

```diff
@@ -83,7 +83,7 @@ export class BasicLightThemeInternal {
   public static controlHeightSmall = '32px';
   public static controlHeightMedium = '40px';
   public static controlHeightLarge = '48px';
-  public static mobileMediaQuery = '(max-width: 576px) and (hover: none) and (pointer: coarse)';
+  public static mobileMediaQuery = '(max-width: 576px) and (pointer: coarse)';
```

Comment:
Если мы пока не отказываемся, то предлагаю вариант, где либо `hover`, либо `pointer`. Так явно будет надежнее:

```suggestion
  public static mobileMediaQuery = '(max-width: 576px) and (hover: none), (pointer: coarse)`';
```

Reply:
@SchwJ 

Собрал демку
https://codepen.io/chabin/pen/ogvqKjY

Reply:
Предлагаю выпилить и `(pointer: coarse)`. Иначе на экранах со стилусами тоже работать не будет. Мобильное выражение в том виде в котором вы его используете должно полагаться только на ширину экрана

Reply:
Мы рассмотрим эту правку для следующего мажора. Задача IF-2149. Если потребуется, переменную темы можно переопределить уже сейчас.